src/app/portlets/dot-edit-page/content/components/dot-edit-content-view-as-toolbar/dot-edit-content-view-as-toolbar.component.spec.ts

```diff
@@ -46,16 +65,19 @@ class MockDotLanguageSelectorComponent {
 }
 
 describe('DotEditContentViewAsToolbarComponent', () => {
+    let componentHost: DotTestHostComponent;
+    let fixtureHost: ComponentFixture<DotTestHostComponent>;
+
     let component: DotEditContentViewAsToolbarComponent;
-    let fixture: ComponentFixture<DotEditContentViewAsToolbarComponent>;
     let de: DebugElement;
-    let languageSelector: DebugElement;
-    let deviceSelector: DebugElement;
-    let personaSelector: DebugElement;
+    let languageSelector: DotLanguageSelectorComponent;
+    let deviceSelector: DotDeviceSelectorComponent;
+    let personaSelector: DotPersonaSelectorComponent;
 
-    beforeEach(() => {
+    beforeEach(async(() => {
```

Comment:
Our component have external template, we need to always use async, more info: https://angular.io/guide/testing#calling-compilecomponents